### DISCUSSION TITLE
API issues - 6/24/2024 ~ 1730 EDT

### DISCUSSION THREAD

**Discussion by aasmith26** [weather-gov/api#737] [General]
Cannot reach any API endpoint currently.  Getting HTTP 503s, have emailed NCO Ops, and the status page is down.

**Comment by aasmith26** [comment]
API is now working.

**Reply by dwhitemv25** [reply]
Looking at my API error log, they flipped API to Boulder somewhere between yesterday and today.

[NCO status page](https://www.nco.ncep.noaa.gov/status/messages/) and NWS Weather Wire-Open Interface are still down as of the date of this reply. NWWS-OI is running out of College Park so that might be where the problems are.

**Reply by dwhitemv25** [reply]
Looking at overnight ADMSDM messages, the network in CP went down. Work has been ongoing to restore services.

NWWS-OI failed over to Boulder and is back up as of 7:30am PDT / 1430Z.
